packages/ember-metal/lib/property_get.ts

```diff
@@ -97,6 +94,16 @@ export function get(obj: object, keyName: string): any {
     typeof keyName !== 'string' || keyName.lastIndexOf('this.', 0) !== 0
   );
   assert('Cannot call `get` with an empty string', keyName !== '');
+  if (
+    (obj as ExtendedObject).isDestroyed &&
```

Comment:
We should modify these `isDestroyed` checks to be using `meta.sourceIsDestroyed` instead...

Reply:
that is how it was done before and same way done in `property_set`, maybe make it later with another PR just to keep things incremental so easier to track things if there is issue ?

Reply:
Yep, totally agreed! I was just trying to point out the issue generally, it definitely isn't related to this PR at all. I'm sorry I didn't mention it in this comment.